docs/FAQ/_index.md

```diff
@@ -46,3 +46,42 @@ We’d love to hear it! Fill out the [submission form](https://aka.ms/LightningG
+
+- **Include the code in the Pull Request (PR)**: You can include the code as part of your PR and host it within the Arc Jumpstart Drops repository.
+- **Keep the code in your own repository**: Alternatively, you can keep the code in your own repository and provide a reference/URL to it in the Drop [definition file](https://github.com/Azure/arc_jumpstart_drops/blob/main/SCHEMA.md).
+
+## Do I need to authenticate to the Arc Jumpstart portal to contribute or use Jumpstart Drops?
+No, authentication and logging in to the portal aren't required to contribute or use Jumpstart Drops. However, if you are logged in, we're developing features to provide contributors with insights on their contributions and enhance their experience and submission.
+
+### What's the Jumnpstart Drop licensing schema?
+Jumpstart Drops follows the same [MIT](https://github.com/Azure/arc_jumpstart_drops/blob/main/LICENSE) licensing as all the other Arc Jumpstart Products. All contributions should also be under an [MIT](https://github.com/Azure/arc_jumpstart_drops/blob/main/LICENSE) license. 
+
+The MIT license is a permissive open-source software license that allows for the use, modification, and distribution of the licensed software, as long as the original copyright notice and license terms are included in any copies or distributions of the software. 
+
+For more information, see [contribution process](../contribution_guidelines/).
+
+## How can I discover a Jumsptart Drop?
+To discover a Jumpstart Drop tailored to your needs, leverage the filters and sorting options available on the Arc Jumpstart Drops main page. Start by visiting [Arc Jumpstart Drops](https://arcjumpstart.com/arc_jumpstart_drops) and then use the panel on the right-hand side to filter by criteria such as *Programming* *Language*, *Product*, *Difficulty*, *Industry*, *Topic*, and more. Additionally, you can always use the search bar at the top to search across all Jumpstart content, including Jumpstart Drops.
+
+## How do I use a Jumpstart Drop?
+Using Jumsptart Drops is straightforward. Simply download the Jumnpstart Drop by clicking the Download button on the Jumpstart Drop's page. After downloading, you will have a .zip file containing all the necessary source code to run the Jumpstart Drop. Depending on the Jumpstart Drop, you may need to configure some prerequisites, but all instructions should be included in the **_index.md** file of the Jumpstart Drop.
+
+## How do I submit a Jumpstart Drop?
+Submitting a Jumpstart Drop is easy. You can choose to create a pull request directly in the Arc Jumpstart Drops repository or use the [Submit Jumpstart Drop](https://arcjumpstart.com/arc_jumpstart_drops) form for a streamlined process. Refer to the Jumpstart Drops [contribution process](../contribution_guidelines/) guide for detailed instructions.
```

Comment:
```suggestion
Submitting a Drop is easy. You can choose to create a pull request (PR) directly in the Arc Jumpstart Drops repository or use the [Submit Jumpstart Drop](https://aka.ms/JumpstartDrop) form for a streamlined process. Refer to the Jumpstart Drops [contribution process](../contribution_guidelines/) guide for detailed instructions.
```

1. "Submit Jumpstart Drop" - shouldn't this be "Create" button/form?
2. Since we are not going live with it, you want to think if you want to incl. this at the moment. I would wait until Create become available.

Reply:
Agree - comenting that part